app/views/master_files/_player.html.erb

```diff
@@ -13,72 +13,22 @@ Unless required by applicable law or agreed to in writing, software distributed
   specific language governing permissions and limitations under the License.
 ---  END LICENSE_HEADER BLOCK  ---
 %>
-<% content_for :page_styles do %>
-  <%= stylesheet_link_tag 'mejs2_player' %>
-<% end %>
-<% content_for :page_scripts do %>
-  <%= javascript_include_tag "mejs2_player" %>
-<% end %>
+<% # Set MediaElement 4 query param variable
+is_mejs4 = params['mejs4'] === 'true' %>
```

Comment:
@adamjarling I pushed the change and flipped the logic so mejs2 is the special case.  Probably doesn't make a big difference but I thought it would be easier when deconstructing mejs2 support.